types/wheel/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for wheel 0.0
```

Comment:
Based on the documentation of the wheel library, the sample usage is using commonjs modules. This dts file would be incompatible. Instead use the "export =" syntax, like:

```javascript
declare function wheel(element: GlobalEventHandlers, callback: (event: WheelEvent) => void, useCapture?: boolean): void;
declare namespace wheel {
    function addWheelListener(element: GlobalEventHandlers,
        callback: (event: WheelEvent) => void,
        useCapture?: boolean): void;

    function removeWheelListener(element: GlobalEventHandlers,
        callback: (event: WheelEvent) => void,
        useCapture?: boolean): void;
}

export = wheel;
```